tutorials/asr/06_Voice_Activiy_Detection.ipynb

```diff
@@ -0,0 +1,1151 @@
+    "\n",
+    "## Download the background data\n",
+    "We suggest to use the background categories of [freesound](https://freesound.org/) dataset  as our non-speech/background data. \n",
+    "We provide scripts for downloading and resampling it.  Please have a look at [NeMo docs VAD Data Preparation]( https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/v0.11.0/voice_activity_detection/tutorial.html#data-preparation). Note that downloading this dataset may takes hours. \n",
+    "\n",
+    "**NOTE:** Here, this tutorial serves as a demonstration on how to train and evaluate models for vad using NeMo. We avoid using freesound dataset, and use `_background_noise_` category in Google Speech Commands Dataset as non-speech/background data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the speech data\n",
+    "   \n",
+    "We will use the open source Google Speech Commands Dataset (we will use V2 of the dataset for the tutorial, but require very minor changes to support V1 dataset) as our speech data. Google Speech Commands Dataset V2 will take roughly 6GB disk space. These scripts below will download the dataset and convert it to a format suitable for use with nemo_asr.\n",
+    "\n",
+    "\n",
+    "**NOTE**: You may additionally pass `--test_size` or `--val_size` flag for spliting train val and test data.\n",
+    "\n",
+    "**NOTE**: You may additionally pass a `--rebalance_method='fixed|over|under'` at the end of the script to rebalance the class samples in the manifest. \n",
+    "* 'fixed': Fixed number of sample for each class. Train 5000, val 1000, and test 1000. (Change number in script if you want)\n",
+    "* 'over': Oversampling rebalance method\n",
+    "* 'under': Undersampling rebalance method\n",
+    "\n",
+    "**NOTE**: The `_background_noise_` category only has 6 audio files. So we would like to generate more based on the audio files to enlarge our background training data. If you want to use your own background noise data, just change the `background_data_root` and delete `--generate`\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tmp = 'src'\n",
+    "data_folder = 'data'\n",
+    "if not os.path.exists(tmp):\n",
+    "    os.makedirs(tmp)\n",
+    "if not os.path.exists(data_folder):\n",
+    "    os.makedirs(data_folder)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script = os.path.join(tmp, 'process_vad_data.py')\n",
+    "if not os.path.exists(script):\n",
+    "    !wget -P $tmp https://raw.githubusercontent.com/NVIDIA/NeMo/candidate/scripts/process_vad_data.py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "speech_data_root = os.path.join(data_folder, 'google_dataset_v2')\n",
```

Comment:
maybe warn that this could take some time

Reply:
actually, how long this should take? running forever on Colab

Reply:
Good catch! It typically won't last long in locally but it running forever on Colab. Suspect it's an I/O issue. Gonna update to have much smaller demo data.

Reply:
@okuchaiev here! to use --demo. need the latest process_vad_data